src/ui/layout/pane.vue

```diff
@@ -78,39 +78,27 @@ export default {
     async mounted() {
         await this.$nextTick();
         // Hide tree and/or inspector pane if specified in URL
-        this.handleHideUrl();
-        this.openmct.router.on('change:params', this.handleHideUrl);
+        if (this.collapsable) {
```

Comment:
Hi @davetsay I think we can get rid of 'collapsable' by checking this.hideParam. Only the 2 panes that has this props are collapsable.

Reply:
changed collapsable from a prop to a computed based on hideParam prop